classes/04class/exercises/c04-iac01/jayanath/iam.tf

```diff
@@ -0,0 +1,51 @@
+# Role to run the EC2
+resource "aws_iam_role" "doa_ec2_runner" {
+  name = "doa_ec2_runner"
+
+  assume_role_policy = <<-EOF
+  {
+    "Version": "2012-10-17",
+    "Statement": [
+      {
+        "Action": "sts:AssumeRole",
+        "Principal": {
+          "Service": "ec2.amazonaws.com"
+        },
+        "Effect": "Allow",
+        "Sid": ""
+      }
+    ]
+  }
+  EOF
```

Comment:
if this gets big, an option is template files: https://medium.com/@mitesh_shamra/terraform-managing-policy-using-templates-e254b2d17907